src/mesh.rs

```diff
@@ -87,6 +88,17 @@ impl Mesh {
 }
 
 impl DynamicMesh {
+    /// Returns the indexed vertex position of the geometry base shape,
+    /// or `None` if the index is out of bounds.
+    pub fn vertex(&self, index: usize) -> Option<mint::Point3<f32>> {
+        self.geometry.base_shape.vertices.get(index).map(Clone::clone)
```

Comment:
`mint` types implement `Copy`, don't they?

Reply:
Yes, but `Option` does not.

Reply:
Option has `cloned()` afaik, it looks much nicer

Reply:
TIL, thanks. :smile:

Reply:
yep, let's just use `cloned` here